### PR TITLE
Add additional keys to release key fingerprints [1.1.1]

### PR DESCRIPTION
**Pull request by t8m**
Added keys for Paul Dale and Tomáš Mráz.

##### Checklist

- [x] documentation is added or updated

### CODE REVIEW

**Comment by t8m** [comment]
ping for the reviews

**Review by mattcaswell — APPROVED** [review]

**Comment by mattcaswell** [comment]
Agree urgent. Should be in the release today.

Ping for a second review

**Comment by t8m** [comment]
@paulidale @levitte could you please approve?

File: doc/fingerprints.txt [hunk 1]
```diff
@@ -22,3 +22,13 @@ pub   2048R/0E604491 2013-04-30
       Key fingerprint = 8657 ABB2 60F0 56B1 E519 0839 D9C4 D26D 0E60 4491
 uid                  Matt Caswell <matt@openssl.org>
 uid                  Matt Caswell <frodo@baggins.org>
+
+pub   rsa3072 2021-02-14
```
Comment (paulidale) [inline doc/fingerprints.txt:26]: This key is old.  It should instead be:
```
pub   rsa4096 2021-02-14 [SC]
      B7C1C14360F353A36862E4D5231C84CDDCC69C45
uid          Paul Dale <pauli@openssl.org>
```

Comment (t8m) [reply doc/fingerprints.txt:26]: Could you please upload it to the keyserver?

**Comment by t8m** [comment]
Updated key for @paulidale included.

**Review by paulidale — APPROVED** [review]

**Comment by t8m** [comment]
@mattcaswell please re-approve. Also @paulidale do you agree with urgent? We want to add it to the release.

**Comment by paulidale** [comment]
Yes, urgent.

**Review by mattcaswell — APPROVED** [review]

**Comment by mattcaswell** [comment]
Repo is currently frozen so it won't appear there yet - but this will be included in today's release.

**Pull request closed without merging**